libs/core/langchain_core/messages/utils.py

```diff
@@ -221,14 +221,14 @@ def _create_message_from_message_type(
         tool_call_id: (str) the tool call id. Default is None.
         tool_calls: (list[dict[str, Any]]) the tool calls. Default is None.
         id: (str) the id of the message. Default is None.
-        **additional_kwargs: (dict[str, Any]) additional keyword arguments.
+        additional_kwargs: (dict[str, Any]) additional keyword arguments.
 
     Returns:
         a message of the appropriate type.
 
     Raises:
         ValueError: if the message type is not one of "human", "user", "ai",
-            "assistant", "system", "function", or "tool".
+            "assistant", "system", "function", "tool", or "developer".
```

Comment:
```suggestion
            "assistant", "function", "tool", "system", or "developer".
```
nit to keep it next to system + match order in error msg